core/src/test/java/ai/timefold/solver/core/api/solver/SolutionManagerTest.java

```diff
@@ -672,14 +940,14 @@ void recommendFitTwoUninitializedEntityWithChained(SolutionManagerSource Solutio
         uninitializedSolution.setChainedEntityList(Arrays.asList(b1, c1, c2, uninitializedEntity, uninitializedEntity2));
 
         var solutionManager = SolutionManagerSource.createSolutionManager(SOLVER_FACTORY_CHAINED);
-        assertThatThrownBy(() -> solutionManager.recommendFit(uninitializedSolution, uninitializedEntity,
+        assertThatThrownBy(() -> solutionManager.recommendAssignment(uninitializedSolution, uninitializedEntity,
```

Comment:
The suggestion to continue testing the `recommendFit` test applies to all instances where we are changing it to `recommendAssignment`.